### PR TITLE
feat: updated Soneium Minato as one of the OP stack chains

### PR DESCRIPTION
**Pull request by impelcrypto**
This PR updates Soneium Minato testnet as one of the OP stack chains

## Detailed summary
* Updated Soneim Minato definitions
  * ref: [Soneium Docs](https://docs.soneium.org/docs/builders/contracts) 
* Added Soneium Minato to OP stack chains

---

## PR-Codex overview
This PR introduces the `Soneium Minato Testnet` to the codebase, updating its configuration, RPC URLs, and associated contracts. It also adds the testnet to relevant files for better integration and accessibility.

### Detailed summary
- Updated `soneiumMinato` to `Soneium Minato Testnet`.
- Changed `nativeCurrency` to `Sepolia Ether`.
- Updated `blockExplorers` name to `Blockscout`.
- Added new contract details for `disputeGameFactory`, `l2OutputOracle`, `portal`, and `l1StandardBridge`.
- Introduced `sourceId` for contract references.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: site/pages/op-stack/chains.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ import {
   zora, // [!code hl]
   zoraSepolia, // [!code hl]
   zoraTestnet, // [!code hl]
+  soneiumMinato, // [!code hl]
```
Comment (jxom) [inline site/pages/op-stack/chains.md:18]: alphabetical order pls

File: src/op-stack/chains.ts [hunk 2]
```diff
@@ -18,3 +18,4 @@ export { snaxTestnet } from '../chains/definitions/snaxTestnet.js'
 export { zora } from '../chains/definitions/zora.js'
 export { zoraSepolia } from '../chains/definitions/zoraSepolia.js'
 export { zoraTestnet } from '../chains/definitions/zoraTestnet.js'
+export { soneiumMinato } from '../chains/definitions/soneiumMinato.js'
```
Comment (jxom) [inline src/op-stack/chains.ts:21]: alphabetical order pls

File: src/chains/definitions/soneiumMinato.ts [hunk 3]
```diff
@@ -1,26 +1,55 @@
...
       url: 'https://explorer-testnet.soneium.org',
-      apiUrl: 'https://explorer-testnet.soneium.org/api/',
+      apiUrl: 'https://explorer-testnet.soneium.org/api',
     },
   },
   contracts: {
+    ...chainConfig.contracts,
+    disputeGameFactory: {
+      [sourceId]: {
+        address: '0xF69dB6cA559C52d9A4BB6e2B2901f490Ca35Fbf6',
```
Comment (jxom) [inline src/chains/definitions/soneiumMinato.ts:28]: is there a source for these contracts (e.g. official docs)?

Comment (jxom) [reply src/chains/definitions/soneiumMinato.ts:28]: dw i see here https://docs.soneium.org/docs/builders/contracts

File: src/op-stack/chains.ts (hunk 2 above)
Comment (impelcrypto) [reply src/op-stack/chains.ts]: Fixed in [this](https://github.com/wevm/viem/pull/2799/commits/dce3303b363187ecbfc6717be0151d391b095745) commit.

File: site/pages/op-stack/chains.md (hunk 1 above)
Comment (impelcrypto) [reply site/pages/op-stack/chains.md]: Fixed in [this](https://github.com/wevm/viem/pull/2799/commits/dce3303b363187ecbfc6717be0151d391b095745) commit.

File: pnpm-lock.yaml
[the remark this answers is not in the document]
Comment (impelcrypto) [reply pnpm-lock.yaml:1]: Fixed in [this](https://github.com/wevm/viem/pull/2799/commits/b293d34e87a849685517462cc656bacf4e47e02d) commit.

File: src/chains/definitions/soneiumMinato.ts (hunk 3 above)
Comment (impelcrypto) [reply src/chains/definitions/soneiumMinato.ts:28]: Yes, we are working on it, [the official docs](https://docs.soneium.org/docs/builders/contracts#minato-testnet-optimism-l1-stack-on-sepolia) will be updated soon to add the 'disputeGameFactory' contract.

**Comment by impelcrypto** [comment]
Hi @jxom thanks for merging this PR. May I know when will you release the new version?

**Pull request merged**